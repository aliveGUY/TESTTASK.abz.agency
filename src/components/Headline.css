.headline {
  background-position: center;
  background-size: cover;
  color: #fff;
  text-align: center;
  padding: 8vh 16px 12vh 16px;
}

.headline-text {
  max-width: 380px;
  margin: 0 auto 30px auto;
}
