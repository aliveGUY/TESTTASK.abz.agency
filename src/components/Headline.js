import React from 'react';
import background from '../img/wheet_field.png';

function Headline() {
  return (
    <div
      className="headline"
      style={{
        backgroundImage: `url(${background})`,
      }}
    >
      <h1 className="headline-text">Test assignment for front-end developer</h1>
      <p className="headline-text">
        What defines a good front-end developer
        is one that has skilled knowledge of HTML,
        CSS, JS with a vast understanding of User design
        thinking as they`ll be building web interfaces with accessibility in mind.
        They should also be excited
        to learn, as the world of
        Front-End Development keeps evolving.
      </p>
      <button type="button" className="Header-nav-usrs">Sign up</button>
    </div>
  );
}
export default Headline;
