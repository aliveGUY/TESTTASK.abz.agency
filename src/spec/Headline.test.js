import renderer from 'react-test-renderer';
import Headline from '../components/Headline';

test('renders headline', () => {
  const tree = renderer.create(<Headline />).toJSON();
  expect(tree).toMatchSnapshot();
});
