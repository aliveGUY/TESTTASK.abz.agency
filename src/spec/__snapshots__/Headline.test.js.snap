// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders headline 1`] = `
<div
  className="headline"
  style={
    Object {
      "backgroundImage": "url(wheet_field.png)",
    }
  }
>
  <h1
    className="headline-text"
  >
    Test assignment for front-end developer
  </h1>
  <p
    className="headline-text"
  >
    What defines a good front-end developer is one that has skilled knowledge of HTML, CSS, JS with a vast understanding of User design thinking as they\`ll be building web interfaces with accessibility in mind. They should also be excited to learn, as the world of Front-End Development keeps evolving.
  </p>
  <button
    className="Header-nav-usrs"
    type="button"
  >
    Sign up
  </button>
</div>
`;
